File: sdks/ruby/rest/lib/Brume/types/api_key_scope.rb
# frozen_string_literal: true

module Brume
  module Types
    module APIKeyScope
      extend Brume::Internal::Types::Enum

      PUBLISH = "publish"
      READ_STATS = "read_stats"
      MANAGE_KEYS = "manage_keys"
      MANAGE_PROJECT = "manage_project"
    end
  end
end

File: sdks/swift/rest/Sources/Schemas/ApiKeyScope.swift
import Foundation

/// API key scope (E6). Legacy keys without explicit scopes keep full
/// access; see migration `007_e_workstream.sql`.
public enum ApiKeyScope: String, Codable, Hashable, CaseIterable, Sendable {
    case publish
    case readStats = "read_stats"
    case manageKeys = "manage_keys"
    case manageProject = "manage_project"
}